src/store/actions/movieDetailsAction.js

```diff
@@ -0,0 +1,16 @@
+import axios from "../axiosHelper";
+
+export const getMovieDetails = (id) => async (dispatch) => {
```

Comment:
An action is an object with "type" and "payload".
A function that returns such objects is an action creator.
A function that delays a call of an action creator is a thunk (async action creator).
So to avoid confusion, "getMovieDetails" as a thunk should be stored under "src/store/thunks".
This "src/store/actions" folder would contain all the simple actions (sync action creators).